Add basic MoveIt configuration package for RX160

gavanderhoorn opened the pull request:
As per subject.

gavanderhoorn committed:
rx160: add basic MoveIt package.

gavanderhoorn committed:
meta: add RX160 MoveIt pkg run_depend.

shaun-edwards commented on staubli_rx160_moveit_config/config/joint_limits.yaml:
```diff
@@ -0,0 +1,31 @@
+  joint_1:
+    has_velocity_limits: true
+    max_velocity: 2.88
+    has_acceleration_limits: false
```
Do we know how this is handled by MoveIt?  I've always set this value to a "realistic" value, otherwise the acceleration was very slow.

gavanderhoorn replied:
'how this is handled'? Do you mean how MoveIt comes up with this value? I've always understood that it just takes a certain percentage of the max velocity limit. I don't have the actual acceleration limits, so I've always left these values like this (but see ros-industrial/ros_industrial_issues#25).

gavanderhoorn replied:
O wait, I misread your comment. I'm actually not sure why there is no value there. All other MoveIt configs I've created (ie, those in `fanuc_experimental`) all had acceleration limits.

gavanderhoorn replied:
Seems that ros-planning/moveit_setup_assistant@0e72040e replaced the old behaviour (set max_accel to 1/5 of the maximum velocity) with a new one, where actual acceleration limits are used (see `src/tools/moveit_config_data.cpp` in that commit). I'm not sure where those limits are supposed to come from though.

gavanderhoorn committed:
rx160: add acceleration limits to MoveIt config.

Until we get hold of the actual limits, use the old setup assistant
approach to calculate some limits (max_vel / 5).

gavanderhoorn commented:
@shaun-edwards: I've added some acceleration limits to the MoveIt config, using the old setup assistant approach (using max_vel/5).

Will update with real values when / if I can find them (see also ros-industrial/ros_industrial_issues#25).

shaun-edwards commented:
+1.
